lib/node_modules/@stdlib/math/base/special/besselj0/lib/besselj0.js

```diff
@@ -7,13 +7,11 @@
 */
 
 /*
-* Copyright John Maddock 2006-7, 2013-14.
-* Copyright Paul A. Bristow 2007, 2013-14.
-* Copyright Nikhar Agrawal 2013-14
-* Copyright Christopher Kormanyos 2013-14
+* Copyright (c) 2007, 2013 John Maddock
+* Copyright Christopher Kormanyos 2013.
 * Use, modification and distribution are subject to the
 * Boost Software License, Version 1.0. (See accompanying file
-* LICENSE or copy at http://www.boost.org/LICENSE_1_0.txt)
+* LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
```

Comment:
LICENSE was correct, I believe. By LICENSE, we are referring to the license file which will be included with the package.

Reply:
Not [this license](https://github.com/boostorg/math/blob/develop/include/boost/math/special_functions/detail/bessel_j0.hpp#L1-L4)?

Reply:
I've reverted it to the original license. Maybe I should just add one extra line:

```
* Copyright (c) 2006 Xiaogang Zhang
```

Reply:
Our LICENSE includes the Boost Software License (see https://github.com/stdlib-js/stdlib/blob/36c0b4134cfbbcc315e3fe6c21ef6c7b4038754c/LICENSE#L181); hence, why we say `LICENSE`, as our license will always ship with individual packages. And yes, the copyright shown in our file should be for Xiaogang Zhang.